File: practica_5/estadistica/estructurada/Estadisticas.java
import java.util.Scanner;

public class Estadisticas {
    
    public static double promedio(double[] numeros) {
        double suma = 0;
        for (double num : numeros) {
            suma += num;
        }
        return suma / numeros.length;
    }

    public static double desviacion(double[] numeros, double promedio) {
        double suma = 0;
        for (double num : numeros) {
            suma += Math.pow(num - promedio, 2);
        }
        return Math.sqrt(suma / (numeros.length - 1));
    }

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        double[] numeros = new double[10];

        System.out.println("Ingrese 10 números:");
        for (int i = 0; i < 10; i++) {
            numeros[i] = sc.nextDouble();
        }
        
        double prom = promedio(numeros);
        double desv = desviacion(numeros, prom);

        System.out.printf("El promedio es %.2f%n", prom);
        System.out.printf("La desviación estándar es %.5f%n", desv);

        sc.close();
    }
}

File: practica_5/estadistica/estructurada/Estadisticas.py
import math

def promedio(numeros):
    return sum(numeros) / len(numeros)

def desviacion(numeros, prom):
    suma = sum((num - prom) ** 2 for num in numeros)
    return math.sqrt(suma / (len(numeros) - 1))

numeros = []

print("Ingrese 10 números:")
for _ in range(10):
    numeros.append(float(input()))

prom = promedio(numeros)
desv = desviacion(numeros, prom)

print(f"El promedio es {prom:.2f}")
print(f"La desviación estándar es {desv:.5f}")
